care/users/reset_password_views.py

```
@@ -223,67 +195,48 @@ def post(self, request, *args, **kwargs):
+                    instance=self,
+                    reset_password_token=token,
                 )
-        # done
+                if not active_user_found and not getattr(
+                    settings, "DJANGO_REST_PASSWORDRESET_NO_INFORMATION_LEAKAGE", False
+                ):
+                    raise exceptions.ValidationError(
+                        {
+                            "username": [
+                                _(
+                                    "There is no active user associated with this username or the password can not be changed"
```

Pylint (3.3.7)

[convention] Line too long (126/100)

(C0301)